vendor.conf

```diff
@@ -5,22 +5,22 @@ github.com/docker/docker 6978a6e25a2e6063f280ec842bd0f3eae99426e1
 github.com/docker/docker-credential-helpers 71779cf7f595bf47f5a56d0629ce044e59b1cffa
 github.com/docker/go d30aec9fd63c35133f8f79c3412ad91a3b08be06
 github.com/docker/go-connections e15c02316c12de00874640cd76311849de2aeed5
-github.com/docker/go-units 651fc226e7441360384da338d0fd37f2440ffbe3
+github.com/docker/go-units v0.3.1
 github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
 github.com/docker/notary v0.4.2
 github.com/gogo/protobuf v0.4
 github.com/gorilla/mux 4c1c3952b7d9d0a061a3fa7b36fd373ba0398ebc
-github.com/opencontainers/go-digest a6d0ee40d4207ea02364bd3b9e8e77b9159ba1eb
-github.com/opencontainers/image-spec f03dbe35d449c54915d235f1a3cf8f585a24babe
+github.com/opencontainers/go-digest 21dfd564fd89c944783d00d069f33e3e7123c448
+github.com/opencontainers/image-spec v1.0.0-rc7
```

Comment:
may as well Go to 1.0.0 here I think, probably no changes anyway, and we are barely using it.